addons/sys_intercom/stringtable.xml

```diff
@@ -74,7 +74,7 @@
         <Key ID="STR_ACRE_sys_intercom_infantryPhone_startCalling">
             <English>Start Calling %1</English>
             <Spanish>Llamar al exterior %1</Spanish>
-            <German>Läuten %1</German>
+            <German>Läute Infanterietelefon %1</German>
```

Comment:
Same as above

Reply:
Assuming this will show as 'Läute `Gegensprechanlage`', reverted